### PR TITLE
DOC-1319 - RS Utilities Reorg: rladmin status page

### PR DESCRIPTION
**Pull request by cmilesb**
Staging: https://docs.redis.com/staging/jira-doc-1319/rs/references/cli-utilities/rladmin/status/

### CODE REVIEW

File: content/rs/references/cli-utilities/rladmin/status.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+        [ sort <column_titles> ]
+        [ issues_only ]
+```
+
+### Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| extra <parameter> | extra options that show more information |
+| sort <column_titles> | sort results by specified column titles |
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/status.md]:
```suggestion
| node <id> | Only show endpoints for the specified node ID |
| extra <parameter> | Extra options that show more information |
| sort <column_titles> | Sort results by specified column titles |
```

File: content/rs/references/cli-utilities/rladmin/status.md [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+
+``` sh
+$ rladmin status extra all
+CLUSTER:
+OK. Cluster master: 1 (198.51.100.2)
+Cluster health: OK, [1, 0.13333333333333333, 0.03333333333333333]
+failures/minute - avg1 1.00, avg15 0.13, avg60 0.03.
+
+CLUSTER NODES:
+NODE:ID ROLE   ADDRESS    EXTERNAL_ADDRESS HOSTNAME     MASTERS SLAVES OVERBOOKING_DEPTH SHARDS CORES FREE_RAM        PROVISIONAL_RAM VERSION   SHA    RACK-ID STATUS
```
Comment (cmilesb) [inline content/rs/references/cli-utilities/rladmin/status.md:59]:
```suggestion
NODE:ID ROLE   ADDRESS      EXTERNAL_ADDRESS HOSTNAME     MASTERS SLAVES OVERBOOKING_DEPTH SHARDS CORES FREE_RAM        PROVISIONAL_RAM VERSION   SHA    RACK-ID STATUS
```

Comment (cmilesb) [inline content/rs/references/cli-utilities/rladmin/status.md:59]:
```suggestion
NODE:ID ROLE   ADDRESS      EXTERNAL_ADDRESS HOSTNAME     MASTERS SLAVES OVERBOOKING_DEPTH SHARDS CORES FREE_RAM        PROVISIONAL_RAM VERSION   SHA    RACK-ID STATUS
```

File: content/rs/references/cli-utilities/rladmin/status.md [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+rladmin status
+        [ extra <parameter> ]
+        [ issues_only]
+```
+
+### Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| extra     | extra options that show more information |
```
Comment (cmilesb) [inline content/rs/references/cli-utilities/rladmin/status.md:29]:
```suggestion
| extra     | Extra options that show more information |
```

File: content/rs/references/cli-utilities/rladmin/status.md [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+        [ extra <parameters> ]
+        [ sort <column_titles> ]
+        [ issues_only ]
+```
+
+### Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| extra <parameter> | extra options that show more information |
```
Comment (cmilesb) [inline content/rs/references/cli-utilities/rladmin/status.md:95]:
```suggestion
| extra <parameter> | Extra options that show more information |
```

File: content/rs/references/cli-utilities/rladmin/status.md [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
+If `issues_only` is specified, only instances that do not have an "OK" status are shown.
+
+### Example
+
+``` sh
+$ rladmin status endpoints
+DB:ID     NAME             ID                    NODE        ROLE        SSL    
+db:1      database1        endpoint:1:1          node:1      single      No     
+db:2      database2        endpoint:2:1          node:2      single      No     
+db:3      database3        endpoint:3:1          node:3      single      N
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/status.md:175]:
```suggestion
db:3      database3        endpoint:3:1          node:3      single      No
```

File: content/rs/references/cli-utilities/rladmin/status.md [hunk 6]
```diff
@@ -0,0 +1,277 @@
...
+        [ sort <column_titles> ]
+        [ issues_only ]
+```
+
+### Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| extra <parameter> | extra options that show more information |
+| sort <column_titles> | sort results by specified column titles |
```
Comment (cmilesb) [inline content/rs/references/cli-utilities/rladmin/status.md:96]:
```suggestion
| sort <column_titles> | Sort results by specified column titles |
```

File: content/rs/references/cli-utilities/rladmin/status.md [hunk 7]
```diff
@@ -0,0 +1,277 @@
...
+        [ extra <parameters> ]
+        [ sort <column_titles> ]
+        [ issues_only ]
+```
+
+### Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| extra <parameter> | extra options that show more information |
```
Comment (cmilesb) [inline content/rs/references/cli-utilities/rladmin/status.md:144]:
```suggestion
| extra <parameter> | Extra options that show more information |
```

File: content/rs/references/cli-utilities/rladmin/status.md (hunk 1 above)
Comment (cmilesb) [inline content/rs/references/cli-utilities/rladmin/status.md:145]:
```suggestion
| sort <column_titles> | Sort results by specified column titles |
```

File: content/rs/references/cli-utilities/rladmin/status.md [hunk 8]
```diff
@@ -0,0 +1,277 @@
...
```
Comment (cmilesb) [inline content/rs/references/cli-utilities/rladmin/status.md:193]:
```suggestion
| extra <parameter> | Extra options that show more information |
```

File: content/rs/references/cli-utilities/rladmin/status.md [hunk 9]
```diff
@@ -0,0 +1,277 @@
...
+| sort <column_titles> | sort results by specified column titles |
```
Comment (cmilesb) [inline content/rs/references/cli-utilities/rladmin/status.md:194]:
```suggestion
| sort <column_titles> | Sort results by specified column titles |
```

File: content/rs/references/cli-utilities/rladmin/status.md [hunk 10]
```diff
@@ -0,0 +1,277 @@
...
+| extra state_machine | Shows execution of state machine information  |
+| extra watchdog | Shows watchdog status  |
+
+### Returns
+
+Returns a table of the status of all shards on the cluster.
+
+If `sort <column_titles>` is specified, the result is sorted by the specified table columns.
+
+If `issues_only` is specified, only instances that do not have an "OK" status are shown.
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/status.md:265]:
```suggestion
If `issues_only` is specified, it only shows shards that do not have an "OK" status.
```

File: content/rs/references/cli-utilities/rladmin/status.md [hunk 11]
```diff
@@ -0,0 +1,277 @@
...
+        [ extra <parameters> ]
+        [ sort <column_titles> ]
+        [ issues_only ]
+```
+
+### Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| extra <parameter> | extra options that show more information |
```
Comment (cmilesb) [inline content/rs/references/cli-utilities/rladmin/status.md:243]:
```suggestion
| extra <parameter> | Extra options that show more information |
```

File: content/rs/references/cli-utilities/rladmin/status.md [hunk 12]
```diff
@@ -0,0 +1,277 @@
...
+| sort <column_titles> | sort results by specified column titles |
```
Comment (cmilesb) [inline content/rs/references/cli-utilities/rladmin/status.md:244]:
```suggestion
| sort <column_titles> | Sort results by specified column titles |
```

File: content/rs/references/cli-utilities/rladmin/status.md [hunk 13]
```diff
@@ -0,0 +1,277 @@
...
+| extra state_machine | Shows execution of state machine information  |
+| extra watchdog | Shows watchdog status  |
+
+### Returns
+
+Returns a table of the status of all databases on the cluster.
+
+If `sort <column_titles>` is specified, the result is sorted by the specified table columns.
+
+If `issues_only` is specified, only instances that do not have an "OK" status are shown.
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/status.md:117]:
```suggestion
If `issues_only` is specified, it only shows databases that do not have an "OK" status.
```

**Review by rrelledge — CHANGES_REQUESTED** [review]
Here are some initial suggestions.

File: content/rs/references/cli-utilities/rladmin/status.md [hunk 14]
```diff
@@ -0,0 +1,280 @@
...
+### Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| extra <parameter> | Extra options that show more information |
+| sort <column_titles> | Sort results by specified column titles |
+| issues_only | Filter out all items that have an "OK" status  |
+
+
+| `extra` parameter | Description |
```
Comment (cmilesb) [inline content/rs/references/cli-utilities/rladmin/status.md:251]:
```suggestion
| Extra parameter | Description |
```

File: content/rs/references/cli-utilities/rladmin/status.md [hunk 15]
```diff
@@ -0,0 +1,280 @@
...
+```
+
+## `status shards`
+
+Displays the current status of all shards on the cluster.
+
+``` sh
+rladmin status shards
+        [ node <id> ] 
+        [ db <db:id | name> ]
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/status.md:236]:
```suggestion
        [ db {db:<id> | <name>} ]
```

File: content/rs/references/cli-utilities/rladmin/status.md [hunk 16]
```diff
@@ -0,0 +1,280 @@
...
+### Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| extra <parameter> | Extra options that show more information |
+| sort <column_titles> | Sort results by specified column titles |
+| issues_only | Filters out all items that have an "OK" status  |
+
+
+| `extra` parameter | Description |
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/status.md:100]:
```suggestion
| Extra parameter | Description |
```

File: content/rs/references/cli-utilities/rladmin/status.md [hunk 17]
```diff
@@ -0,0 +1,280 @@
...
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/status.md:150]:
```suggestion
| Extra parameter | Description |
```

File: content/rs/references/cli-utilities/rladmin/status.md [hunk 18]
```diff
@@ -0,0 +1,280 @@
...
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/status.md:199]:
```suggestion
| Extra parameter | Description |
```

**Review by rrelledge — CHANGES_REQUESTED** [review]
Looks good! Here's a few more small suggestions.

File: content/rs/references/cli-utilities/rladmin/status.md [hunk 19]
```diff
@@ -0,0 +1,280 @@
...
+        [ extra <parameters> ]
+        [ sort <column_titles> ]
+        [ issues_only ]
+```
+
+### Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| extra <parameter> | Extra options that show more information |
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/status.md:145]:
```suggestion
|-----------|-------------|
| node <id> | Only show endpoints for the specified node ID |
| extra <parameter> | Extra options that show more information |
```

File: content/rs/references/cli-utilities/rladmin/status.md [hunk 20]
```diff
@@ -0,0 +1,280 @@
...
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/status.md:246]:
```suggestion
|-----------|-------------|
| node <id> | Only show shards for the specified node ID |
| db db:<id> | Only show shards for the specified database ID |
| db <name> | Only show shards for the specified database name |
| extra <parameter> | Extra options that show more information |
```

**Review by rrelledge — CHANGES_REQUESTED** [review]
I spotted a couple of options missing from the parameter tables.

**Review by rrelledge — APPROVED** [review]
Looks good to me.

**Pull request merged**